src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/MLDsa/MLDsaTestHelpers.cs

```diff
@@ -20,6 +20,9 @@ internal static partial class MLDsaTestHelpers
         // TODO: Windows does not support signing empty data. Remove this and use MLDsa.IsSupported (or remove condition) when it does.
         internal static bool SigningEmptyDataIsSupported => MLDsa.IsSupported && !PlatformDetection.IsWindows;
 
+        // TODO: Windows does not support external mu. Remove this and use MLDsa.IsSupported (or remove condition) when it does.
```

Comment:
I would recommend leaving it in, even if it is the same as `MLDsa.IsSupported`. macOS does not have external mu, so we will just end up re-introducing it again.

So I would just remove this comment.